server/routers/work-shares/activateWallet.ts

```diff
@@ -116,9 +116,12 @@ export const activateWallet = protectedProcedure
     const walletPublicKey = userWallet.publicKey;
 
     if (!userWallet || userWallet.status !== WalletStatus.ENABLED) {
-      throw new TRPCError({
+      throw new TRPCError(userWallet ? {
```

Comment:
Should not it be `!userWallet ?`here and in others as well ?

Reply:
True! 😅